src/commands/create_manifest.ts

```diff
@@ -25,17 +27,20 @@ new CLICommand({
 		WalletFileParameter,
 		SeedPhraseParameter,
 		...ConflictResolutionParams,
-		...TreeDepthParams
+		...TreeDepthParams,
+		GatewayParameter
 	],
 	action: new CLIAction(async function action(options) {
 		const parameters = new ParametersHelper(options, cliWalletDao);
 
 		const wallet = await parameters.getRequiredWallet();
+		const arweave = getArweaveFromURL(parameters.getGateway());
 
 		const arDrive = cliArDriveFactory({
 			wallet: wallet,
 			feeMultiple: parameters.getOptionalBoostSetting(),
-			dryRun: !!options.dryRun
+			dryRun: !!options.dryRun,
```

Comment:
should use `isDryRun` function

Reply:
[352bfc1](https://github.com/ardriveapp/ardrive-cli/pull/246/commits/352bfc1f91ab4ea0c59ec96cd90d89f0eb9496a9)